docs/UserGuide.md

```diff
@@ -153,6 +153,20 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Clearing all entries : `view`
```

Comment:
I think the header here should be something like "View an existing contact's details"?